### PR TITLE
🌐(frontend) fix various translations

### PR DESCRIPTION
**Pull request by NathanVss**
## Purpose

- [x] fix unwanted spaces
- [x] deal with "An error occurred while address {action}"

### CODE REVIEW

**Review by jbpenrath — APPROVED** [review]
Looks good to {action} ✅

File: src/frontend/js/components/DashboardAddressesManagement/DashboardEditAddress.spec.tsx [hunk 1]
```diff
@@ -169,6 +169,6 @@ describe('<DashboardEditAddress/>', () => {
     // An error banner is shown.
     const banner = container!.querySelector('.banner--error') as HTMLElement;
     expect(banner).not.toBeNull();
-    await findByText(banner!, 'An error occurred: Internal Server Error. Please retry later.');
+    await findByText(banner!, 'An error occurred while address update. Please retry later.');
```
Comment (sampaccoud) [inline src/frontend/js/components/DashboardAddressesManagement/DashboardEditAddress.spec.tsx:172]:
```suggestion
    await findByText(banner!, 'An error occurred while updating the address. Please retry later.');
```

File: src/frontend/js/components/DashboardAddressesManagement/index.spec.tsx [hunk 2]
```diff
@@ -351,6 +351,6 @@ describe('<DashAddressesManagement/>', () => {
     // It shows an error banner.
     const banner = container!.querySelector('.banner--error') as HTMLElement;
     expect(banner).not.toBeNull();
-    await findByText(banner!, 'An error occurred: Internal Server Error. Please retry later.');
+    await findByText(banner!, 'An error occurred while addresses fetching. Please retry later.');
```
Comment (sampaccoud) [inline src/frontend/js/components/DashboardAddressesManagement/index.spec.tsx:354]:
```suggestion
    await findByText(banner!, 'An error occurred while fetching addresses. Please retry later.');
```

File: src/frontend/js/hooks/useAddresses.ts [hunk 3]
```diff
@@ -10,11 +10,25 @@ import { useSessionMutation } from 'utils/react-query/useSessionMutation';
...
 const messages = defineMessages({
-  error: {
-    id: 'hooks.useAddresses.error',
-    description:
-      'Error message shown to the user when address creation/update/deletion request fails.',
-    defaultMessage: 'An error occurred: {error}. Please retry later.',
+  errorUpdate: {
+    id: 'hooks.useAddresses.errorUpdate',
+    description: 'Error message shown to the user when address update request fails.',
+    defaultMessage: 'An error occurred while address update. Please retry later.',
```
Comment (sampaccoud) [inline src/frontend/js/hooks/useAddresses.ts:16]:
```suggestion
    defaultMessage: 'An error occurred while updating the address. Please retry later.',
```

File: src/frontend/js/hooks/useAddresses.ts [hunk 4]
```diff
@@ -10,11 +10,25 @@ import { useSessionMutation } from 'utils/react-query/useSessionMutation';
...
+  },
+  errorDelete: {
+    id: 'hooks.useAddresses.errorDelete',
+    description: 'Error message shown to the user when address deletion request fails.',
+    defaultMessage: 'An error occurred while address deletion. Please retry later.',
```
Comment (sampaccoud) [inline src/frontend/js/hooks/useAddresses.ts:21]:
```suggestion
    defaultMessage: 'An error occurred while deleting the address. Please retry later.',
```

File: src/frontend/js/hooks/useAddresses.ts [hunk 5]
```diff
@@ -10,11 +10,25 @@ import { useSessionMutation } from 'utils/react-query/useSessionMutation';
...
+  },
+  errorCreate: {
+    id: 'hooks.useAddresses.errorCreate',
+    description: 'Error message shown to the user when address creation request fails.',
+    defaultMessage: 'An error occurred while address creation. Please retry later.',
```
Comment (sampaccoud) [inline src/frontend/js/hooks/useAddresses.ts:26]:
```suggestion
    defaultMessage: 'An error occurred while creating the address. Please retry later.',
```

File: src/frontend/js/hooks/useAddresses.ts [hunk 6]
```diff
@@ -10,11 +10,25 @@ import { useSessionMutation } from 'utils/react-query/useSessionMutation';
...
+  },
+  errorSelect: {
+    id: 'hooks.useAddresses.errorSelect',
+    description: 'Error message shown to the user when addresses fetch request fails.',
+    defaultMessage: 'An error occurred while addresses fetching. Please retry later.',
```
Comment (sampaccoud) [inline src/frontend/js/hooks/useAddresses.ts:31]:
```suggestion
    defaultMessage: 'An error occurred while fetching addresses. Please retry later.',
```

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**